main.py

```diff
@@ -278,4 +301,87 @@ def default_out_parser(metadata_dict, filename, metadata_category):
   fileobj.close()
 
 
+def get_command_line_args():
+  """
+  <Purpose>
+    Uses the argparse module to parse, process and return the command line 
+    arguments.
+
+  <Arguments>
+    None.
+
+  <Exceptions>
+    TBD.
+
+  <Return>
+    An object representing the command line arguments.
+  """
+
+  parser = argparse.ArgumentParser(prog='main.py', description='Captures the given build/test command\'s I/O and relevant system details.')
+  parser.add_argument('--version', action='version', version=TOTO_TOOL_VERSION)
+  parser.add_argument('--input', metavar='FILEPATH', help='the path to the desired input file to be routed through stdin')
+  parser.add_argument('--policy', metavar='FILENAME', help='the path to the desired file specifying build/test policy')
+  parser.add_argument('command', metavar='COMMAND', type=str, help='the bash command to execute the build or test')
+
+  return parser.parse_args()
+
+
+def process_policy_file(metadata_dict, policy_filepath):
```

Comment:
I think it is fine to have an exception thrown, the user can always bypass by setting the field as null.  Ex: if they want to run on different OS, etc.